grpc/src/compiler/cpp_generator.cc

```diff
@@ -1155,13 +1155,15 @@ grpc::string GetSourcePrologue(grpc_generator::File *file,
 
     vars["filename"] = file->filename();
     vars["filename_base"] = file->filename_without_ext();
+    vars["message_header_ext"] = file->message_header_ext();
```

Comment:
what do these do?

Reply:
Prior to #6954 the `.grpc.fb.cc` file was including the message generated `.h`. 
For some reason, i see that this exact change is now braking [this](https://github.com/google/flatbuffers/runs/4551484783?check_suite_focus=true) test.